<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KFDAPanel</class>
 <widget class="QWidget" name="FisherKernelPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>373</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>371</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="train_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>85</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Train</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eval_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>85</width>
     <height>28</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="text">
    <string>Evaluate</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Options:</string>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>86</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>RBF</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Kernel:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Training samples each class:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="samples">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dimension after PCA:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dimension">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose a photo to classify:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="photo_dir">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_photo">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>40</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="photo_label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>121</width>
     <height>161</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="classify_button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>85</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Classify</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
